libraries/postgres/README.md

```diff
@@ -300,6 +302,75 @@ The binary payload is derived by first base64-decoding each `__bufN_val` propert
 That binary payload, in turn, is defined in the [`EncryptedBaseType` constructor](https://github.com/taskcluster/azure-entities/blob/c6f63e3553c71f0859a5d6338ce5e7c7eb8c9671/src/entitytypes.js#L716-L725).
 It is the concatenation of a 128-bit (16-byte) random initialization vector (`iv`) and the ciphertext produced by aes-256 in CBC mode.
 
+## Signing
+
+Taskcluster signs database rows, generally including all columns in the row.
+The signature takes the form of a "signature container", similar to a "crypto container", a JSONB object with properties `kid`, `v`, and `ser`.
+The `kid` field references signing keys (a distinct set from crypto keys), and the versions in `v` are numbered distinctly from crypto containers.
+This container is typically stored in a `signature` column.
+
+The versions `v` encompass algorithm and encoding, and are described below.
+
+The `ser` property, short for "serialization", is an integener defining the columns that are included in the serialized input to the signature algorithm.
+When a table migration occurs that adds a column to the signature, a new `ser` must be introduced, with the old version continuing to verify the old data.
```

Comment:
Is there a test that would fail if someone forgets to do this or is that something each service needs to test individually?

Reply:
We could do so if we require that signatures always cover the entire row, and not just a subset of the columns.  That's always been the case for Azure, so seems reasonable to enforce here.  Thanks!

Reply:
For now, I'll include the requirement that all rows be considered in the docs.  For anything we're porting from Azure, that's already the case.  I bumped this improvement out to #3180.  We can do that if we decide to keep signing support.